### ISSUE TITLE
Art Improvement.Perma Opened thread

### ISSUE THREAD

**Issue by Alarcarr** [Alarcarr/Rising-Star#17]
Removed

**Comment by Alarcarr** [comment]
Better just have a temporal art folder or tryouts folder to choose later

**Comment by darloth** [comment]
Perhaps a little unrelated, but.. I liked it, but I thought the solar panel textures should all be oriented the same way around.

**Comment by Alarcarr** [comment]
Yeah I think that too , its a materr of setting the UV maps correcty in 3ds but at such low zoom its difficult to noticed in game if u want the proyect files to toy with the model just ask..

**Comment by Alarcarr** [comment]
I reopened the thread in the hopes of more discussions.

**Comment by Alarcarr** [comment]
The weird one (radial is live) xD

![solarcell](https://cloud.githubusercontent.com/assets/10220609/15376981/b20c9bb6-1d58-11e6-8a30-242443b96bf6.png)
